inc/functions_upload.php

```diff
@@ -313,6 +313,11 @@ function upload_avatar($avatar=array(), $uid=0)
 		case "image/x-png":
 			$img_type = 3;
 			break;
+		case "image/bmp":
+		case "image/x-bmp":
+		case "image/x-bitmap":
+			$img_type = 4;
```

Comment:
@Stefan-ST , they're listed under `IMAGETYPE` not `IMAGE`. I also had 6 when I tested.